src/migrations/20190120141524-review.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.createTable('Reviews', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER
+            },
+            BookId: {
+                allowNull: false,
+                type: Sequelize.INTEGER,
+                references: {
+                    model: 'Books',
+                    key: 'id'
+                }
+            },
+            UserId: {
+                allowNull: false,
+                type: Sequelize.INTEGER,
+                references: {
+                    model: 'Users',
+                    key: 'id'
+                }
+            },
+            Rating: {
+                allowNull: false,
+                type: Sequelize.INTEGER
+            },
+            Text: {
+                defaultValue: false,
+                allowNull: false,
+                type: Sequelize.STRING
```

Comment:
What does this translate down to in PostgreSQL? Is it a VARCHAR? Because if so, this will have a 255 character limit.